apps/daimo-bot/api/index.ts

```diff
@@ -0,0 +1,33 @@
+import bodyParser from "body-parser";
+import dotenv from "dotenv";
+import express from "express";
+import { WebhookEvent } from "./types";
+import { PaymentActionProcessor } from "./PaymentActionProcessor";
+
+dotenv.config();
+
+const app = express();
+const port = 4000;
+
+app.use(bodyParser.json());
+app.get("/", (req, res) => res.send("Up!"));
```

Comment:
copy `/health` from daimo-api for consistency

Reply:
Sounds good, though fwiw I prefer "/" so you can verify on deploys that server is up in-browser with less effort—sometimes I forget healthcheck endpoints per codebase and have to look em up